### DISCUSSION TITLE
dev environment with authentication webhook URL

### DISCUSSION THREAD

**Discussion by hugbubby** [hasura/graphql-engine#5816] [General]
I sometimes run the app that handles hasura authentication webhook requests on my local machine, for testing. Unfortunately, I can't change the URL hasura uses to make those requests programmatically, so for now I have to push the app into the dev environment where it will receive such requests every time I want to test.

**Comment by esseswann** [comment]
This sounds like a general problem of microservices development. Try either developing your hasura also locally or tools like https://www.telepresence.io/
